pkg/lockfile/lockfile_unix.go

```diff
@@ -35,9 +35,9 @@ type lockfile struct {
 // necessary.
 func openLock(path string, ro bool) (int, error) {
 	if ro {
-		return unix.Open(path, os.O_RDONLY, 0)
+		return unix.Open(path, os.O_RDONLY|unix.FD_CLOEXEC, 0)
```

Comment:
Shouldn't this be `unix.O_CLOEXEC`?

Reply:
Yes, I originally looked for os.O_CLOEXEC, and later found unix.FD_CLOEXEC.  Did not look for O_CLOEXEC